rkt/image_list.go

```diff
@@ -78,6 +80,54 @@ var (
 	}
 )
 
+type printedImage struct {
+	ID         string `json:"id"`
+	Name       string `json:"name"`
+	ImportTime string `json:"import_time"`
+	LastUsed   string `json:"last_used"`
+	Size       string `json:"size"`
+}
+
+func (r *printedImage) attributes() []string {
+	return []string{r.ID, r.Name, r.Size, r.ImportTime, r.LastUsed}
+}
+
+func (r *printedImage) printableString() string {
+	return strings.Join(r.attributes(), "\t")
+}
+
+type outputFormat int
```

Comment:
One last nit... Typically the consts for an "enum like thing" are grouped right under the type definition (for example, [ConnState in the go stdlib]](https://github.com/golang/go/blob/7179c1acd967daea44fb806865e57308ea0c3679/src/net/http/server.go#L2442-L2480)).